sanic/request.py

```diff
@@ -86,11 +86,15 @@ def token(self):
 
         :return: token related to request
         """
+        prefixes = ('Token ', 'Bearer ')
```

Comment:
micro optimization I know, but Bearer being more common, shouldn't it be first ?